### PR TITLE
0.11.0 changelog tweaks

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

The changelog has grown in size and is becoming more and more unreadable. A bit of structure would help.

### Reviewer guidance

?

### References

#4390 

----

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
+ - Fix: Allow usernames in non-latin alphabets
+ - Fix: Drive listing and space availability reporting
+ - Auto-refresh in coach reports
  - Added more validation to help with log-in
- - Improvements in drive listing and space availability reporting
- - Improved rendering of some pages on smaller screens
  - Security: upgraded Python cryptography and pyopenssl libraries for CVE-2018-10903
- - Many additional bug fixes, under-the-hood updates, and performance improvements
+
+Translations
```
Comment (rtibbles) [inline CHANGELOG.rst]: Maybe internationalization and localization?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+~~~~~~~~~~~~
+
  - Languages: [ coming soon... ]
+ - Improved consistency of language across the application, and renamed "Superuser" to "Super admin"
+ - Many fixes to internationalization
```
Comment (rtibbles) [inline CHANGELOG.rst]: Many fixes to translation and localization?

Comment (benjaoming) [reply CHANGELOG.rst]: Alright, corrected... am a bit confused about the distinction between internationalization, translation and localization at this point.. but please have a look at the current state :)

**Review by rtibbles — APPROVED** [review]

**Comment by indirectlylit** [comment]
nice update, thanks

**Pull request merged**